libs/common-services/src/main/java/uk/gov/di/ipv/core/library/dto/BackEndCriConfig.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.di.ipv.core.library.dto;
+
+import com.nimbusds.jose.jwk.ECKey;
+import lombok.Getter;
+
+import java.net.URI;
+import java.text.ParseException;
+
+@Getter
+public class BackEndCriConfig implements CriConfig {
```

Comment:
I think my personal taste is `Backend` rather than `BackEnd`, but I'll get over it

Reply:
Yeah, that looks a bit nicer, happy to change it out.